README.md

```diff
@@ -41,27 +43,24 @@ Here are some examples of what you can do with ownCloud Web:
 While the `web` frontend provides a performant, elegant, accessible and themeable base, it also aims to be extendable with custom extensions provided by external developers.
 
 ## Live Demos
+
 - **Infinitescale** - Try the latest commit on the master branch with an ownCloud Infinite Scale backend:
-  - Demo: [https://ocis.ocis-web.latest.owncloud.works](https://ocis.ocis-web.latest.owncloud.works)
+  - Demo: [https://demo.owncloud.com/](https://demo.owncloud.com/)
   - User: `einstein`
```

Comment:
Let's also drop the user as it is mentioned on the demo page itself?

![image](https://github.com/user-attachments/assets/7e05055d-ffb6-4061-a0b9-e2b99b535ba3)